hermes-api/src/main/java/pl/allegro/tech/hermes/api/SchemaMetadata.java

```diff
@@ -0,0 +1,62 @@
+package pl.allegro.tech.hermes.api;
+
+import com.google.common.base.MoreObjects;
+
+import java.util.Objects;
+
+public final class SchemaMetadata {
```

Comment:
In my opinion it is not correct name for this class as it contains not only metadata but also raw schema. In another comment I suggested `RawSchemaWithMetadata`. WDYT?

Reply:
I suggested by confluence response implementation: https://github.com/confluentinc/schema-registry/blob/master/client/src/main/java/io/confluent/kafka/schemaregistry/client/SchemaMetadata.java. But name: `RawSchemaWithMetadata` it's alsow okey for me.